### DISCUSSION TITLE
Dim Reg Epsilon convention used by PaXEvaluate

### DISCUSSION THREAD

**Discussion by gp4895** [FeynCalc/feyncalc#289] [Q&A]
Hi, 
I am wondering what convention is used for D when evaluating integrals using PaXEvaluate and how this can be changed. Specifically I would like to use D=4+2EpsilonIR and D=4+2EpsilonUV when using PaXEvaluateUVIRSplit.

**Comment by vsht** [comment]
Hi,

the dimension is always fixed to D=4-2*ep because it is hard-coded in 
Package-X this way.



Am 05.12.24 um 11:29 schrieb gp4895:
